release_images.yml

```diff
@@ -3,50 +3,51 @@ release_images:
-      force_release: False
-  3:
-    framework: "tensorflow"
-    version: "2.7.0"
-    customer_type: "ec2"
-    arch_type: "graviton"
+      example: False        # [Default: False] Set to True to denote that this image is an Example image
+      disable_sm_tag: False # [Default: False] Set to True to prevent SageMaker Abbreviated Tags from being attached
+      # to images being published.
+      force_release: False   # [Default: False] Set to True to force images to be published even if the same image
+      # has already been published. Re-released image will have minor version incremented by 1.
     inference:
-      device_types: ["cpu"]
+      device_types: [ "cpu", "gpu" ]
       python_versions: [ "py38" ]
       os_version: "ubuntu20.04"
+      cuda_version: "cu113"
       example: False        # [Default: False] Set to True to denote that this image is an Example image
-      disable_sm_tag: True # [Default: False] Set to True to prevent SageMaker Abbreviated Tags from being attached
+      disable_sm_tag: False # [Default: False] Set to True to prevent SageMaker Abbreviated Tags from being attached
       # to images being published.
-      force_release: False  # [Default: False] Set to True to force images to be published even if the same image
+      force_release: False   # [Default: False] Set to True to force images to be published even if the same image
       # has already been published. Re-released image will have minor version incremented by 1.
-  4:
+  2:
     framework: "pytorch"
-    version: "1.5.1"
+    version: "1.12.1"
     arch_type: "x86"
+    customer_type: "ec2"
+    training:
+      device_types: [ "cpu", "gpu" ]
+      python_versions: [ "py38" ]
+      os_version: "ubuntu20.04"
+      cuda_version: "cu113"
+      example: False        # [Default: False] Set to True to denote that this image is an Example image
+      disable_sm_tag: False # [Default: False] Set to True to prevent SageMaker Abbreviated Tags from being attached
+      # to images being published.
+      force_release: False   # [Default: False] Set to True to force images to be published even if the same image
+      # has already been published. Re-released image will have minor version incremented by 1.
     inference:
-      device_types: ["eia"]
-      python_versions: ["py38"]
+      device_types: [ "cpu", "gpu" ]
+      python_versions: [ "py38" ]
       os_version: "ubuntu20.04"
-      example: False
-      disable_sm_tag: True
+      cuda_version: "cu113"
+      example: False        # [Default: False] Set to True to denote that this image is an Example image
+      disable_sm_tag: False # [Default: False] Set to True to prevent SageMaker Abbreviated Tags from being attached
+      # to images being published.
+      force_release: False   # [Default: False] Set to True to force images to be published even if the same image
+      # has already been published. Re-released image will have minor version incremented by 1.
```

Comment:
Add configuration for example image as well like https://github.com/aws/deep-learning-containers/blob/master/.release_images_template.yml#L929